plyer/platforms/ios/proximity.py

```diff
@@ -0,0 +1,37 @@
+'''
+iOS Proximity
+-------------
+'''
+
+from pyobjus import autoclass
+from pyobjus.dylib_manager import load_framework
+from plyer.facades import Proximity
+
+load_framework('/System/Library/Frameworks/UIKit.framework')
+UIDevice = autoclass('UIDevice')
+
+
+class iOSProximity(Proximity):
+
+    def __init__(self):
+        super(iOSProximity, self).__init__()
+        self.device = UIDevice.currentDevice()
+
+    def _enable(self):
+        self.device.setProximityMonitoringEnabled_(True)
+
+    def _disable(self):
+        self.device.setProximityMonitoringEnabled_(False)
+
+    def _get_proximity(self):
+        if self.device.proximityMonitoringEnabled:
+            if self.device.proximityState:
+                return str(True)
```

Comment:
Why now returning a string of a boolean, both platforms are coded to return a boolean value anyway?